core/storage/base_model/gae_models_test.py

```diff
@@ -133,57 +131,20 @@ def test_put(self):
             base_models.BaseModel.get_by_id(model_id).last_updated)
         last_updated = model.last_updated
 
-        # Field last_updated won't get updated because update_last_updated_time
-        # is set to False and last_updated already has some value.
-        model.update_timestamps(update_last_updated_time=False)
-        model.put()
-        self.assertEqual(
-            base_models.BaseModel.get_by_id(model_id).last_updated,
-            last_updated)
-
         # Field last_updated will get updated because update_last_updated_time
         # is set to True (by default).
-        model.update_timestamps()
         model.put()
         self.assertNotEqual(
             base_models.BaseModel.get_by_id(model_id).last_updated,
-            last_updated)
-
-    def test_put_without_update_timestamps(self):
```

Comment:
Here & below: do the removed test behaviors make sense for the human-maintained model to still verify as correct?

Reply:
Not sure what you mean here? I have added tests for the BaseHumanMaintainedModel to verify that the function stays ± the same.

Reply:
Hmm, reading the tests here more closely it seems like they are specifically fixed on update_timestamps and I suppose aren't relevant anymore.